src/routes/help/_categories.js

```diff
@@ -1,20 +1,20 @@
 const categories = [
   {
     slug: "get-started",
-    title: "Get Started",
-    text: "Learn how to get started with Cal-Adapt",
+    title: "Getting Started",
```

Comment:
Did we settle on Getting Started or Get Started? I don't remember. The phrase might be used in couple of different locations, like blog posts or events or Help sections.

Reply:
Oh right, I think we ended up settling on "Get Started" though I feel like in certain places "Getting Started" seems to read a little better? I'll change it back to "Get Started"